File: mpp/LabMPPCurse/Models/src/main/java/Models/Destinatie/Destinatie.java
package Models.Destinatie;

import Models.HasId;

public class Destinatie implements HasId<Integer> {

    private int id;
    private String nume;

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer integer) {
        this.id=integer;
    }

    public Destinatie(int id, String nume) {
        this.id = id;
        this.nume = nume;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    @Override
    public String toString() {
        return "Destinatie{" +
                "id=" + id +
                ", nume='" + nume + '\'' +
                '}';
    }
}

File: mpp/CurseMPP/CurseMPP/Models/Destinatie/Destinatie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CurseMPP.Models
{
	public class Destinatie : HasId<int>
	{	

		//private static int increment = 0;

		public Destinatie(int id, string nume)
		{
			Id = id;
			this.nume = nume;
		}

		public int Id { get; set; }
		public string nume { get; set; }

		public override string ToString()
		{
			return "Id: " + Id + "  Nume:  " + nume;
		}
	}
}
